### PR TITLE
vcr release draft post

### PR DESCRIPTION
**Pull request by sckott**
Hey @steffilazerte - can you take a look at this technote?

### CODE REVIEW

**Comment by sckott** [comment]
thanks!

**Comment by maelle** [comment]
Uninvited comments from me 😁 What a big release!

- Maybe the section about the new ctb could come at the end, as it's less important for users. 🙂
- The conclusion could mention the book again (read the vcr vignettes, and the book for more context?)
- could you add examples of pretty and non pretty cassettes, and maybe link to a package using the JSON serializer? Also say how to convert a package to the new serializer: update the vcr config and delete the YAML cassettes?

**Comment by maelle** [comment]
reg rmdhunks maybe link (in a footnote?) to
* https://www.garrickadenbuie.com/blog/dry-vignette-and-readme/
* https://blog.r-hub.io/2019/12/03/readmes/#tools-for-writing-and-re-using-content (it mostly repeats what Garrick wrote but it also mentions roxygen2's `@includeRmd` that was used in vcr)
These are really useful tips.

**Comment by sckott** [comment]
Thanks @maelle ! I should have cc'ed you to make sure you saw this. 

Well, i think the new contributor is important, lets see what Steffi says. 

Will mention book again, can show pretty in brief at least. I don't have any pkgs using the JSON serializer for everything, but I think i have one test at least. And rmdhunks links

**Comment by steffilazerte** [comment]
Did you want to make any changes or are you ready for review? 😉

**Comment by sckott** [comment]
will do now

**Comment by sckott** [comment]
@steffilazerte  okay

**Review by steffilazerte — COMMENTED** [review]
Hi @sckott,

Looks good, just a couple of typos to fix, a wording suggestion or two, fix the date, and put the title in title case. 

Sorry @maelle, I agree with Sckott that the New Contributor part makes sense right at the beginning :grin: It's an important part of package development and it's also important to ensure everyone remembers that packages are developed by **people**. Besides, knowing that you're a major contributor can only help boost vcr's reputation 😉 

* [X] post follows Content Guidelines
* [X] post follows Style Guide
* [x] title is in Title Case
* [ ] publication date is ok
    * **@stefaniebutland what date is this one for?**
* [X] alternative text of images is informative
* [X] Twitter metadata looks ok (paste post preview link in [Twitter card validator](https://cards-dev.twitter.com/validator); might have to click twice on "Preview card")
* [X] author metadata is provided with correct folder name
* [X] html not included in pull request of Rmd post
* [X] I ran `roblog::ro_lint_md()` on index.md
* [X] I ran `roblog::ro_check_urls()` on index.md
* [X] I ran a spell-check on index.md
* [X] YAML subject tags are ok

File: content/technotes/2020-12-17-vcr-release/index.md [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+
+## New contributor
+
+vcr has a new author: [Maëlle Salmon](https://ropensci.org/author/ma%C3%ABlle-salmon/). She has been [contributing to the package quite a lot lately](https://github.com/ropensci/vcr/graphs/contributors), including many documentation improvements and big updates to the [HTTP Testing in R][book] which covers vcr in addition to some other packages.
+
+## New serializer
+
+vcr now has two serializer options: YAML (which we had before) and JSON (the new one). Serializing here refers to what format the HTTP request and response are put in before being written to a file. If you're a keen observer of these two data formats, you'll know these are not that different, with YAML a superset of JSON. The clear advantage of JSON straight away is that it avoids the common problem with vcr enabled tests on Windows platform where using YAML leads to errors due to something about the `yaml` package. The `jsonlite` package, which does the heavy lifting when using JSON, does not suffer from the same problems on Windows.
```
[the remark this answers is not in the document]
Comment (sckott) [reply content/technotes/2020-12-17-vcr-release/index.md:26]: i hope so too

File: content/technotes/2020-12-17-vcr-release/index.md [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+output: 
+  html_document:
+    keep_md: true
+---
+
+A new version of [vcr][] was just released. See the [release notes](https://github.com/ropensci/vcr/releases/tag/v0.6.0) for all the details. I want to highlight a few of the more noteable changes. 
+
+## New contributor
+
+vcr has a new author: [Maëlle Salmon](https://ropensci.org/author/ma%C3%ABlle-salmon/). She has been [contributing to the package quite a lot lately](https://github.com/ropensci/vcr/graphs/contributors), including many documentation improvements and big updates to the [HTTP Testing in R][book] which covers vcr in addition to some other packages.
```
[the remark this answers is not in the document]
Comment (sckott) [reply content/technotes/2020-12-17-vcr-release/index.md:22]: woops, thanks

File: content/technotes/2020-12-17-vcr-release/index.md [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+
+This inject/eject workflow has already been used for [caching HTTP requests in vignettes](https://github.com/ropensci/vcr/issues/150).
+
+## Better debugging experience
+
+There is a new vcr article [debugging](https://docs.ropensci.org/vcr/articles/debugging) that helps guide you through debugging tests that use vcr. Paths work sightly different when running tests hitting a button in RStudio or running `devtools::test()` vs. running tests line by line in an interactive R session. The article includes discussion of a new function `vcr::vcr_test_path()` - based on `testthat::test_path()` - which will set the correct path to vcr files whether you are in an interactive R session or not. The article also includes guidance on logging to help in your debugging process.
+
+## Documentation
+
+Many documentation improvements were made, much of it by Maëlle. Repeated parts of documentation were put into `man/rmdhunks` ([^1], [^2]) to be used in various places of the packages documentation. A new debugging vignette was created - see above section. In addition, a separate vignette was parsed out covering internals.
```
[the remark this answers is not in the document]
Comment (sckott) [reply content/technotes/2020-12-17-vcr-release/index.md:134]: yes, will do

**Comment by steffilazerte** [comment]
I'm not sure if you wanted another review, but I took a quick look and everything looks good except the date. :grin:

**Comment by sckott** [comment]
@steffilazerte  We never got a publication date from Stef though, correct?

**Pull request merged**